### PR TITLE
Fix camera stream on frontend and recognition service startup

### PR DESCRIPTION
**Pull request by Mala1180**
- Fix env variables on frontend
- Add docker build stage to Recognition
- Fix env variables on Recognition
- Add admitted token in alarm
- Fix camera webrtc stream on frontend (modified mediamtx configuration)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -137,6 +137,7 @@ gradle-app.setting
 
 # Python
 venv
```
Comment (paga16-hash) [inline .gitignore:139]: maybe i'm wrong, it is still necessary?

**Review by paga16-hash — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (Mala1180) [reply .gitignore:139]: no, we'll keep .venv, so venv is unnecessary

Comment (paga16-hash) [reply .gitignore:139]: ok right, you can remove it

File: alarm/src/index.ts [hunk 2]
```diff
@@ -22,9 +22,9 @@ const server: HttpServer = http.createServer(app)
 
 app.use((req: Request, res: Response, next: NextFunction) => {
   const authHeader = req.headers.authorization
-  const token = authHeader && authHeader.split(' ')[1]
+  const token = authHeader && authHeader.split(' ')[1] || ''
 
-  if (token === process.env.DEV_API_KEY) return next()
+  if (jwtManager.admittedTokens().includes(token)) return next()
```
Comment (paga16-hash) [inline alarm/src/index.ts:27]: after the if to check if the token is one of the admitted you have to put 
  if (token === undefined || token === '') return res.status(403).send({ error: 'No authentication token' })
  (like in other middlewares)
  to check the token. Adding this line the test that now fail in CI should pass.

**Pull request merged**